From 8e2b8cde9f1ea759d0317a2c69266c77b5411f5d Mon Sep 17 00:00:00 2001
From: Rahul T R <r-ravikumar@ti.com>
Date: Tue, 25 Jul 2023 21:53:20 +0530
Subject: [PATCH] arm64: dts: ti: k3-j721e-sk-rpi-hdr-ehrpwm: Set the updated
 pinmux for gpio0

Pinmux is updated for main_gpio0 in the overlay, but not set. Fix this
by setting the updated pinmux to main_gpio0

Signed-off-by: Rahul T R <r-ravikumar@ti.com>
---
 arch/arm64/boot/dts/ti/k3-j721e-sk-rpi-hdr-ehrpwm.dtso | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/ti/k3-j721e-sk-rpi-hdr-ehrpwm.dtso b/arch/arm64/boot/dts/ti/k3-j721e-sk-rpi-hdr-ehrpwm.dtso
index fa4f0c99a73f..2de85bbf7e63 100644
--- a/arch/arm64/boot/dts/ti/k3-j721e-sk-rpi-hdr-ehrpwm.dtso
+++ b/arch/arm64/boot/dts/ti/k3-j721e-sk-rpi-hdr-ehrpwm.dtso
@@ -61,3 +61,8 @@
 	pinctrl-0 = <&rpi_header_ehrpwm3_pins_default>;
 	status = "okay";
 };
+
+&main_gpio0 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&rpi_header_gpio0_pins_default>;
+};
-- 
2.17.1

